Libs/MRML/Core/vtkMRMLPlotChartNode.cxx

```diff
@@ -142,6 +117,21 @@ void vtkMRMLPlotChartNode::WriteXML(ostream& of, int nIndent)
 {
   // Start by having the superclass write its information
   Superclass::WriteXML(of, nIndent);
```

Comment:
Example of using macros for simplifying MRML node property read/write XML, copy, and printself.
@jcfr @pieper @cpinter

Reply:
I like the macros a lot 👍